examples/Toast.order.jsx

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { useToaster, Button } from '@itwin/itwinui-react';
+
+export default () => {
+  const toaster = useToaster();
+
+  React.useEffect(() => {
+    toaster.setSettings({
+      order: 'ascending',
```

Comment:
Similarly, we could showcase all `order` values.

Reply:
I have added a `Select` for orders.